### ISSUE TITLE
Using NSS, PAM, ... for LDAP connection handling and information gathering

### ISSUE THREAD

**Issue by Oline** [Turanic/usbwall#2]
Instead of maintening your own LDAP connection for getting information about the users, maybe it could be nice to use an existing system service to get the LDAP information for you.

As entry point, you could look at that presentation about NSS, PAM and SSSD interactions: https://jhrozek.fedorapeople.org/presentations/linuxdays/linuxdays-2012-sssd.pdf

**Comment by Turanic** [comment]
I agree, that would be a good feature. I put this in the **todo** list.
